frontend/src/app/shared/helpers/drag-and-drop/drag-and-drop.service.ts

```diff
@@ -4,6 +4,7 @@ import {
 import { DOCUMENT } from '@angular/common';
 import { DomAutoscrollService } from 'core-app/shared/helpers/drag-and-drop/dom-autoscroll.service';
 import { findIndex, reinsert } from 'core-app/shared/helpers/drag-and-drop/drag-and-drop.helpers';
+import dragula from 'dragula';
```

Comment:
Need to double-check this - Dragula is required with the `expose-loader` in `init-vendors.ts`, which we're not currently importing in `test.ts`. We are also not importing `typings/shims.d.ts`.